src/database/accessors/loadBundle.ts

```diff
@@ -10,12 +9,10 @@ export const loadBundle = async <T extends object>(
   tinyPgQueryName: string,
   tinyPgQueryParameters: TinyPgParams,
   tableName: string,
-  entryValidator: ValidateFunction<T>,
 ): Promise<Bundle<T>> => {
-  const entries = await loadObjects(
+  const entries = await loadObjects<T>(
```

Comment:
I think only reason for `loadObjects` outside of `loadBundle` is if we were using an enrichment pattern.  I'm not sure we have need to load objects outside of a bundle context if we are going with the "one complex query per type of object selection" approach.